src/runner/index.js

```diff
@@ -123,16 +123,26 @@ export default class Runner extends EventEmitter {
     _validateRunOptions () {
         const concurrency = this.bootstrapper.concurrency;
         const speed       = this.opts.speed;
-        const proxyBypass = this.opts.proxyBypass;
+        let proxyBypass   = this.opts.proxyBypass;
 
         if (typeof speed !== 'number' || isNaN(speed) || speed < 0.01 || speed > 1)
             throw new GeneralError(MESSAGE.invalidSpeedValue);
 
         if (typeof concurrency !== 'number' || isNaN(concurrency) || concurrency < 1)
             throw new GeneralError(MESSAGE.invalidConcurrencyFactor);
 
-        if (proxyBypass)
-            assertType(is.string, null, '"proxyBypass" argument', proxyBypass);
+        if (proxyBypass) {
+            assertType([ is.string, is.array ], null, '"proxyBypass" argument', proxyBypass);
```

Comment:
What if something like
```
{ proxyBypass: ['inbox.google.com,mail.google.com','mail.yandex.ru,passport.yandex.ru']}
```
would be passed to the method?

Reply:
there will be 2 rules
1)inbox.google.com,mail.google.com
2)mail.yandex.ru,passport.yandex.ru
which do not have sense, so requests will go through the proxy